src/natcap/invest/hra.py

```diff
@@ -2457,10 +2457,11 @@ def _get_info_dataframe(base_info_table_path, file_preprocessing_dir,
         raise ValueError('Info table %s is not a CSV nor an Excel file.' %
                          base_info_table_path)
 
-    # Convert column names to upper case, and encode it to strings if it's not
-    # a string (e.g. unicode)
+    # Convert column names to upper case, strip whitespace,
+    # and encode it to strings if it's not a string (e.g. unicode)
     info_df.columns = [
-        col.upper() if isinstance(col, str) else col.encode('utf-8').upper()
+        col.strip().upper() if isinstance(col, str) 
+        else col.encode('utf-8').upper()

```

Comment:
This `.encode` stuff reminds me of how we used to have to handle encodings for python2/python3 cross-compatibility.  Now that `natcap.invest` is python3-only, perhaps the `.encode` stuff here can be removed now that all `str` objects are unicode?

Reply:
I would think so, all the column names should be `str`